### PR TITLE
Move Risk Tracker changelog to page on website

### LINKED ISSUE #762: Move the Risk Tracker changelog onto the website
Currently, it lives here: https://docs.google.com/document/d/1iwTFoCS8lOIWWm-ZzcMZ_mPHgA8tHVVA3yhKY23gDu8/edit#

Move it to the website so it's tracked as a code change each time.

### PR DESCRIPTION
**Pull request by blanchardjeremy**
The Risk Tracker changelog currently lives here as a Google Doc: https://docs.google.com/document/d/1iwTFoCS8lOIWWm-ZzcMZ_mPHgA8tHVVA3yhKY23gDu8/edit#

This PR moves it to the codebase so it lives on the website. 

Closes #762

### CODE REVIEW

File: src/posts/tracker/changelog.ts [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+interface Change {
+  date: Date
+  versionNum?: string
+  title?: string
+  spreadsheetURL?: string
+  whatsNew: string
+  instructions?: string
+}
+
+// NOTE: Do not include '/edit' at the end of the URL, because we append different paths like /copy or /template/preview depending on the use case. Do not include a slash at the end of the URL.
```
Comment (beshaya) [inline src/posts/tracker/changelog.ts:15]: Consider adding a test to verify this (or opening an issue to add a test)

**Review by beshaya — APPROVED** [review]
LGTM % add a test or open an issue for comment.

Also, looks like you used <pre><code> instead of ```? If you're having trouble with the backticks ending the string, you need to escape them, like \`\`\`. I don't insist.

File: src/posts/tracker/changelog.ts (hunk 1 above)
Comment (blanchardjeremy) [reply src/posts/tracker/changelog.ts:15]: Do you mean a test in the test suite or a "test" in this file that runs every time the code runs and cleans the formatting if it was entered incorrectly?

**Comment by blanchardjeremy** [comment]
@beshaya as for `<pre><code>` vs \`\`\` --- For some reason our markdown generator turns three ticks into <code> instead of <pre><code> ... so 🤷‍♂️

**Pull request merged**